Tools/Scripts/webkitpy/common/system/executive.py

```diff
@@ -376,11 +376,13 @@ def run_command(self,
                     pass_fds=()):
         """Popen wrapper for convenience and to work around python bugs."""
         assert(isinstance(args, list) or isinstance(args, tuple))
-        start_time = time.time()
+        start_time = time.monotonic()
 
         stdin, string_to_communicate = self._compute_stdin(input)
         stderr = self.STDOUT if return_stderr else None
 
+        _log.debug(f'Running "{self.command_for_printing(args)}"...')
```

Comment:
Rather than calling `self.command_for_printing(args)` multiple times, we should just save the result in some variable; elsewhere I've gone with `repro_cmd`, but maybe in this case just `cmd_str` or something?

Reply:
Good point! I'll do that.